src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/InstanceRepository.cs

```diff
@@ -1,21 +1,19 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Net;
-using System.Threading.Tasks;
-using Altinn.Platform.Storage.Configuration;
-using Altinn.Platform.Storage.Models;
-using Microsoft.Azure.Documents;
-using Microsoft.Azure.Documents.Client;
-using Microsoft.Azure.Documents.Linq;
-using Microsoft.Extensions.Options;
-using Newtonsoft.Json;
-using Serilog;
-
 namespace Altinn.Platform.Storage.Repository
```

Comment:
namespacedeclaration should be after all using statements

Reply:
ok